.github/workflows/full-ci.yml

```diff
@@ -52,6 +52,45 @@ jobs:
       - name: Integration tests
         run: mage integrationfull
 
+  full-ci-ce-linux-arm64:
+    if: |
+      (github.event_name == 'push') ||
+      (github.event_name == 'pull_request' &&
+        github.event.action == 'labeled' &&
+        github.event.label.name == 'full-ci') ||
+      (github.event_name == 'pull_request' &&
+        github.event.action == 'synchronize' &&
+        contains(github.event.pull_request.labels.*.name, 'full-ci'))
+    runs-on: graviton
+    strategy:
+      fail-fast: false
+    container:
+      image: tarantool/testing:tt-build
+      options: '--init --privileged'
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+          submodules: recursive
+
+      - name: Install test requirements
+        run: |
+          pip3 install setuptools==59.5.0
+          pip3 install -r test/requirements.txt
+        shell: bash
+
+      - name: Install etcd
+        uses: ./.github/actions/setup-etcd
+
+      - name: Build test
+        run: mage build
+
+      - name: Unit tests
+        run: mage unitfullskipdocker
```

Comment:
Please, make it sub-command like for `coverage`, `default` and `full`. Not done for integration yet, but can be done for unit.